app/services/signbank_export.rb

```diff
@@ -10,22 +10,38 @@ class URLForbiddenCharacterError < StandardError; end
       signs.secondary,
       signs.description,
       signs.notes,
-      array_to_string(array_agg(DISTINCT topics.name), '#{SEPARATOR}') AS topic_names,
-      array_to_string(array_agg(DISTINCT videos.id), '#{SEPARATOR}') AS videos,
-      array_to_string(array_agg(DISTINCT illustrations.id), '#{SEPARATOR}') AS illustrations,
-      array_to_string(array_agg(DISTINCT usage_examples.id), '#{SEPARATOR}') AS usage_examples
+      signs.created_at,
+      contributors.email AS contributor_email,
+      contributors.username AS contributor_username,
+      COUNT(DISTINCT agrees.id) AS agrees,
+      COUNT(DISTINCT disagrees.id) AS disagrees,
+      ARRAY_TO_STRING(ARRAY_AGG(DISTINCT topics.name), '#{SEPARATOR}') AS topic_names,
+      ARRAY_TO_STRING(ARRAY_AGG(DISTINCT videos.id), '#{SEPARATOR}') AS videos,
+      ARRAY_TO_STRING(ARRAY_AGG(DISTINCT illustrations.id), '#{SEPARATOR}') AS illustrations,
+      ARRAY_TO_STRING(ARRAY_AGG(DISTINCT usage_examples.id), '#{SEPARATOR}') AS usage_examples,
+      ARRAY_TO_STRING(
+        ARRAY_AGG(
+          DISTINCT CONCAT_WS(': ',
+          CASE sign_comments.anonymous WHEN true THEN 'Anonymous' ELSE sign_commenters.username END,
+          sign_comments.comment)), '#{SEPARATOR}'
+        ) AS sign_comments
     FROM
       signs
       FULL OUTER JOIN sign_topics ON sign_topics.sign_id = signs.id
+      FULL OUTER JOIN users AS contributors ON contributors.id = signs.contributor_id
       FULL OUTER JOIN topics ON sign_topics.topic_id = topics.id
+      FULL OUTER JOIN sign_comments ON signs.id = sign_comments.sign_id AND sign_comments.removed = false AND sign_comments.display = true
```

Comment:
```suggestion
      FULL OUTER JOIN sign_comments ON signs.id = sign_comments.sign_id AND sign_comments.removed = false AND sign_comments.display = true AND sign_comments.folder_id IS NULL
```

Comments can be made in the context of a folder, and shouldn't be included here either.